test/test-pages/wikipedia/expected.html

```diff
@@ -13,7 +13,7 @@
             </tr>
             <tr>
                 <th scope="row">Founded</th>
-                <td>February 28, 1998<span>; 18 years ago</span>
+                <td>February&#160;28, 1998<span>; 18 years ago</span>
```

Comment:
I don't know, and I'm sure my code shouldn't do anything to make this happen.

Just to make sure I checkout to the master branch and generate test case for `wikipedia`, and true enough the same thing happened.

Reply:
Hm, fair. I was confused about why this currently passes tests, seems it's because `test-readability.js` collapses consecutive whitespace (as HTML does) to avoid issues where wrapping differences between nodes cause differences that won't be visible when the document is rendered. It does this by replacing `\s+` (which will match non-breaking spaces) to a single plain space. Not sure how best to address that... Anyway, not your problem right now.